File: 058.LengthOfLastWord/lastWord.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        i = len(s) - 1
        while i >= 0:
            if s[i] != ' ':
                break
            i -= 1
        count = 0
        while i >= 0 and s[i] != ' ':
            count += 1
            i -= 1

        return count

if __name__ == '__main__':
    print Solution().lengthOfLastWord("    ")

File: 058.LengthOfLastWord/lastWord.cc
#include <string>
#include <iostream>

using namespace std;

class Solution {
public:
    int lengthOfLastWord(string s) {
        int i = int(s.size()) - 1;
        while (i >= 0) {
            if (s[i] != ' ') {
                break;
            }
            i--;
        }

        int count = 0;
        while (i >= 0 && s[i] != ' ') {
            count++;
            i--;
        }

        return count;
    }
};

int main(int argc, char const *argv[]) {
    Solution s = Solution();
    cout << s.lengthOfLastWord("   ") << endl;
    return 0;
}
